src/core/lib/layer.js

```diff
@@ -535,9 +535,13 @@ export default class Layer {
         () => `dataChanged: ${flags.dataChanged} in ${this.id}`);
     }
     if (flags.updateTriggersChanged && !changeFlags.updateTriggersChanged) {
-      changeFlags.updateTriggersChanged = flags.updateTriggersChanged;
+      changeFlags.updateTriggersChanged =
+        changeFlags.updateTriggersChanged && flags.updateTriggersChanged ?
```

Comment:
Maybe this condition can be simplified? We are inside an `if (flags.updateTriggersChanged &&`...